netbout-client/src/main/java/com/netbout/mock/MkAttachment.java

```diff
@@ -117,6 +120,44 @@ public boolean unseen() {
     }
 
     @Override
+    public Date date() throws IOException {
+        try {
+            return new JdbcSession(this.sql.source())
+                .sql("SELECT date FROM attachment WHERE bout = ? AND name = ?")
+                .set(this.bout)
+                .set(this.label)
+                .select(
+                    new Outcome<Date>() {
+                        @Override
+                        public Date handle(
+                            final ResultSet rset,
+                            final Statement stmt
+                        ) throws SQLException {
```

Comment:
@CarlosGines we do not use paired bracket notation to method declarations, let's join lines